corehq/apps/reports/commtrack/standard.py

```diff
@@ -272,7 +268,7 @@ def rows(self):
             row_dict = dict(self.product_dict, **dict(loc_data))
             yield [loc_name] + [
                 v if v is not None else _('No data')
-                for k, v in sorted(row_dict.items(), key=lambda(k, v): k)]
+                for k, v in sorted(row_dict.items(), key=lambda(k, v): self.product_names[k])]
```

Comment:
in this case `row_dict` is a mapping between product_id to the value of that product, like 50, 60, etc, so we'd be sorting by product value not product name